Carosa.Core/_context.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Carosa.Core
+{
+    public class _context : DbContext
```

Comment:
Please name public classes with PascalCase. This one can be named like DatabaseContext. 
Create a directory called "Database" and put it in it.